docs/pages/elevation.js

```diff
@@ -0,0 +1,166 @@
+// @flow strict
```

Comment:
Can/should this also include guidance on Box's docs as well? At a minimum the relevant prop descriptions should link to this page; ideally there would be specific guidance for those props as well.

Reply:
I'll make a separate ticket for this